doc/Makefile

```diff
@@ -218,14 +210,6 @@ $(MANPAGES_BUILDDIR) $(MANPAGES_WEBDIR_LINUX) $(MANPAGES_WEBDIR_WINDOWS):
 
 groff: $(MANPAGES_7_GROFF) $(MANPAGES_5_GROFF) $(MANPAGES_3_GROFF) $(MANPAGES_1_GROFF)
 
-doxygen_docs:
-	doxygen cppobj.Doxyfile
-
-html: $(HTMLFILES) doxygen_docs
```

Comment:
I think only doxygen_docs should be removed

Reply:
Done.